core/tests/test_utils.py

```diff
@@ -350,16 +389,30 @@ def set_moderators(self, moderator_usernames):
             config_domain.MODERATOR_USERNAMES, moderator_usernames)
 
     def get_current_logged_in_user_id(self):
+        """ Gets the user_id of the current logged-in user.
+
+        Returns:
+            str. Returns USER_ID from the environment variable.
```

Comment:
str. The user_id of the currently logged-in user.

(Document interface, not implementation. The client doesn't care where the user id stuff is persisted.)